src/components/views/delegateProfile/DelegateProfile.js

```diff
@@ -7,10 +7,6 @@ import { setActivePage } from '../../../actions';
 import { bindActionCreators } from 'redux';
 
 class DelegatesProfile extends Component {
-  constructor(props) {
```

Comment:
I think i will get a merge errors because i extended the constructor to fetch the profile of a delegate

Reply:
@tomploem okay then I'll revert this change :)